<launch>
    
    <!-- Spawn Task-2 Models in Gazebo -->
    <include file="$(find pkg_vb_sim)/launch/task5_world.launch" />

    <!-- Spawn two UR5 in Gazebo -->
    <include file = "$(find pkg_vb_sim)/launch/two_ur5_gazebo.launch" />

    <!-- Spawn T5 Boxes -->
    <rosparam file ="$(find pkg_vb_sim)/config/config_package_colour.yaml"/>
    <node name= "task5_spawn_models" pkg= "pkg_vb_sim" type="task5_spawn_models.py"/>

    <!-- Run Conveyor Belt Service Server -->
    <node name= "node_service_server_conveyor_belt" pkg= "pkg_vb_sim" type="node_service_server_conveyor_belt.py" output="screen"/>

    <!-- Run the Online Order Placer Node -->
    <rosparam file ="$(find pkg_vb_sim)/config/config_online_order.yaml"/>
    <node name= "node_online_order_placer" pkg= "pkg_vb_sim" type="node_online_order_placer.py" output="screen"/>

    <rosparam command="load" file="$(find pkg_ros_iot_bridge)/config/config_pyiot.yaml"/>
    <node name="node_action_server_ros_iot_bridge" pkg="pkg_ros_iot_bridge" type="node_action_server_ros_iot_bridge.py" output="screen"/>
    <node name="node_detect_pkgs" pkg="pkg_task5" type="order_recive.py" output="screen"/>
	
    <rosparam command="load" file="$(find pkg_task5)/config/order.yaml"/>
<!--     <node name="node_detect_pks" pkg="pkg_task5" type="test.py" output="screen"/> -->
    <include file = "$(find pkg_vb_sim)/launch/two_ur5_move_group.launch" />
</launch>